graph-apps/graph-apps.adoc

```diff
@@ -0,0 +1,675 @@
+= Graph Apps for Neo4j Desktop
+:slug: graph-apps
+:level: Intermediate
+:section: Graph Apps
```

Comment:
Isn't that ecosystem?

Reply:
I thought it could sit in a new section (/developer/graph-apps/), that way as the docs grow they are all under the same URL.  Happy to change it though if you think that would be better?